frontend/src/pages/Community.vue

```diff
@@ -0,0 +1,218 @@
+            :pageSize="pageSize"
+          />
+        </b-tab>
+        <b-tab :title="$t('navigation.community')">
+          <contribution-list
+            :items="itemsAll"
+            @update-list-contributions="updateListAllContributions"
+            @update-contribution-form="updateContributionForm"
+            :contributionCount="contributionCountAll"
+            :showPagination="true"
+            :pageSize="pageSizeAll"
+          />
+        </b-tab>
+      </b-tabs>
+    </div>
+  </div>
+</template>
+<script>
+import ContributionForm from '@/components/Contributions/ContributionForm.vue'
+import ContributionList from '@/components/Contributions/ContributionList.vue'
+import { createContribution, updateContribution, deleteContribution } from '@/graphql/mutations'
+import { listContributions, listAllContributions, verifyLogin } from '@/graphql/queries'
+
+export default {
+  name: 'Community',
+  components: {
+    ContributionForm,
+    ContributionList,
+  },
+  data() {
+    return {
+      tabIndex: 0,
+      items: [],
+      itemsAll: [],
+      currentPage: 1,
+      pageSize: 25,
+      pageSizeAll: 25,
+      contributionCount: 0,
+      contributionCountAll: 0,
+      form: {
+        id: null,
+        date: '',
+        memo: '',
+        amount: '',
+      },
+    }
+  },
+  methods: {
+    setContribution(data) {
+      this.$apollo
+        .mutate({
+          fetchPolicy: 'no-cache',
+          mutation: createContribution,
+          variables: {
+            creationDate: data.date,
+            memo: data.memo,
+            amount: data.amount,
+          },
+        })
+        .then((result) => {
+          this.toastSuccess(result.data)
+          this.updateListContributions({
+            currentPage: this.currentPage,
+            pageSize: this.pageSize,
+          })
+          this.verifyLogin()
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    updateContribution(data) {
+      this.$apollo
+        .mutate({
+          fetchPolicy: 'no-cache',
+          mutation: updateContribution,
+          variables: {
+            contributionId: data.id,
+            creationDate: data.date,
+            memo: data.memo,
+            amount: data.amount,
+          },
+        })
+        .then((result) => {
+          this.toastSuccess(result.data)
+          this.updateListContributions({
+            currentPage: this.currentPage,
+            pageSize: this.pageSize,
+          })
+          this.verifyLogin()
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    deleteContribution(id) {
+      this.$apollo
+        .mutate({
+          fetchPolicy: 'no-cache',
+          mutation: deleteContribution,
+          variables: {
+            id: id,
+          },
+        })
+        .then((result) => {
+          // console.log('result', result.data)
+          this.toastSuccess(result.data)
```

Comment:
Please toast a message and not the result.

Reply:
ok